examples/isolated-objects/isolated_objects.bal

```diff
@@ -0,0 +1,86 @@
+import ballerina/io;
+
+type Coordinates record {|
+    decimal latitude;
+    decimal longitude;
+|};
+
+isolated class Cities {
+
+    final string country;
+
+    // The initialization expression of a field of an
+    // `isolated` object has to be an `isolated` expression,
+    // which itself will be an isolated root.
+    // e.g., expressions with an immutable static type,
+    // simple literals, constructor expressions
```

Comment:
```suggestion
    // simple literals and constructor expressions
```

Reply:
We have "etc." at the end, is this change correct?